OpenKh.Kh2/Battle/Btlv.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using System.IO;
+using Xe.BinaryMapper;
+
+namespace OpenKh.Kh2.Battle
+{
+    public class Btlv
+    {
+        [Data] public int Id { get; set; }
+        [Data] public int ProgressFlag { get; set; }
+        [Data] public byte WorldZZ { get; set; }
+        [Data] public byte WorldOfDarkness { get; set; }
+        [Data] public byte TwilightTown { get; set; }
+        [Data] public byte DestinyIslands { get; set; }
+        [Data] public byte HollowBastion { get; set; }
+        [Data] public byte BeastCastle { get; set; }
+        [Data] public byte OlympusColiseum { get; set; }
+        [Data] public byte Agrabah { get; set; }
+        [Data] public byte LandOfDragons { get; set; }
+        [Data] public byte HundredAcreWoods { get; set; }
+        [Data] public byte PrideLands { get; set; }
+        [Data] public byte Atlantica { get; set; }
+        [Data] public byte DisneyCastle { get; set; }
+        [Data] public byte TimelessRiver { get; set; }
+        [Data] public byte HalloweenTown { get; set; }
+        [Data] public byte WorldMap { get; set; }
+        [Data] public byte PortRoyal { get; set; }
+        [Data] public byte SpaceParanoids { get; set; }
+        [Data] public byte TheWorldThatNeverWas { get; set; }
+        [Data(Count = 5)] public byte[] Padding { get; set; }
```

Comment:
Is this stated as padding in the decomp? It's not 0s.
![image](https://user-images.githubusercontent.com/20492864/116454019-d000b980-a85f-11eb-9441-85de878bdbed.png)

Reply:
![grafik](https://user-images.githubusercontent.com/29046361/117041970-b225cf80-ad0b-11eb-8f7c-a5666716a8ba.png)